llvm-spirv/test/extensions/INTEL/SPV_INTEL_function_pointers/alias.ll

```diff
@@ -1,5 +1,3 @@
-; XFAIL: *
```

Comment:
Why this change was done? The test is not `XFAIL`ed in the upstream translator

Reply:
I'm not sure why the test was `XFAIL`ed in intel/llvm, I believe that it's related to the fact that several patches were missing (e.g. the ones I'm restoring by this PR). That's why we are removing the `XFAIL` here to be aligned with the upstream translator.